elements/designer-mini-nav/designer-mini-nav.html

```diff
@@ -47,6 +47,46 @@
 
   Polymer({
     is: 'designer-mini-nav',
+
+    properties: {
+
+      expanded: {
+        type: Boolean,
+        value: false,
+        reflectToAttribute: true,
+      },
+
+      _collapsing: {
+        type: Boolean,
+        value: function() { return false; },
+      },
+
+    },
+
+    listeners: {
+      'designer-mini-nav-collapse': '_onCollapse',
```

Comment:
My point is that it still needs some other bit of info to differentiate between elements that are draggable within the collapsable thing, and therefore shouldn't trigger a collapse, and those that a draggable out of the collapsable thing, and therefore should trigger a collapse.

In this case a one off solution is quite efficient, and the coupling is pretty loose. If one of use tries to use the element palette outside the mini-nav, the worst thing that will happen is a spurious event.